src/services/NodeMonitor.ts

```diff
@@ -54,10 +57,10 @@ export class NodeMonitor {
 			this.isRunning = true;
 			this.clear();
 
-			await this.getNetworkType(); // Read network Type from provided nodes.
+			await this.getNetworkInfo(); // Read network Type and generated hash seed from provided nodes.
```

Comment:
Should we rename this method as it's not a getter method (doesn't return anything)?
How about `fetchAndSetNetworkInfo`?